main.py

```diff
@@ -1,15 +1,17 @@
 from modules import Clients, ServerWithGCI, GCI
 import simpy
 
-SIM_DURATION = 10
+SIM_DURATION = 12
 env = simpy.Environment()
 
-gci = GCI(env)
-server = ServerWithGCI(env, gci)
+server = ServerWithGCI(env)
 requests = list()
 clients = Clients(env, server, requests)
 
 env.run(until=SIM_DURATION)
 
+print("heap %.10f" % server.heap.level)
 print("remaining request in queue: %i" % len(server.queue.items))
-print("Processed value %.3f" % requests[0]._processed_time)
\ No newline at end of file
+print("processed requests %.3f" % len(requests))
```

Comment:
Shouldn't this be %i instead of %.3f? Lenght is an integer, right?

Reply:
Yes, indeed. I was debugging it before and was using floats even for integers. I'll fix it too.

Reply:
Done, @danielfireman could you review?